### PR TITLE
Add runbot_skip_tests module

### PR DESCRIPTION
**Pull request by bwrsandman**
Module simply takes out --test-enable from command line when spawning
processes

### CODE REVIEW

File: runbot_skip_tests/runbot_build.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        """Remove "--test-enable from cmd line"""
+        cmd = [c for c in cmd if c != '--test-enable']
+        return self.spawn(
+            cmd,
+            lock_path,
+            log_path,
+            cpu_limit=cpu_limit,
+            shell=shell,
+            showstderr=showstderr,
+        )
```
Comment (gurneyalex) [inline runbot_skip_tests/runbot_build.py:43]: just asking, I have not checked the code in the "parent" class: would it not be nicer to call super().spawn after editing `cmd`?

Comment (bwrsandman) [inline runbot_skip_tests/runbot_build.py:43]: Good catch

**Comment by bwrsandman** [comment]
Updated

**Comment by max3903** [comment]
:+1:

File: runbot_skip_tests/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    'summary': 'Skip tests on runbot builds',
+    'version': '1.0',
+    'description': """
+Runbot Skip Tests
+=================
+
+This module will take out "--tests-enable" from all commands given to instances
+built by runbot.
+
+Speeds up runbot if the only intention is to do browsing tests
```
Comment (lepistone) [inline runbot_skip_tests/__openerp__.py]: maybe "manual" testing is more clear, but maybe that's just me.

File: runbot_skip_tests/runbot_build.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class runbot_build(orm.Model):
+    _inherit = "runbot.build"
+
+    def spawn(self, cmd, lock_path, log_path, cpu_limit=None, shell=False,
+              showstderr=False):
+        """Remove "--test-enable from cmd line"""
```
Comment (lepistone) [inline runbot_skip_tests/runbot_build.py]: missing `"`

**Comment by lepistone** [comment]
I like the idea of using runbot for manual testing only. Thanks! :+1:

File: runbot_skip_tests/runbot_build.py (hunk 3 above)
Comment (bwrsandman) [inline runbot_skip_tests/runbot_build.py]: Good catch. What's annoying is that I saw that too but I thought I had fixed it.

Unclosed quotes are "the worst.

File: runbot_skip_tests/runbot_build.py (hunk 1 above)
Comment (bwrsandman) [inline runbot_skip_tests/runbot_build.py:43]: Done

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**